plugins/modules/fabric_transits_workflow_manager.py

```diff
@@ -0,0 +1,1354 @@
+                if not fabric_transit_details:
+                    self.msg = (
+                        "The SDA fabric transit config is not created with the config: {config}"
+                        .format(config=item)
+                    )
+                    self.status = "failed"
+                    return self
+
+                if fabric_transit_details.get("type") == "IP_BASED_TRANSIT":
+                    fabric_transit_index += 1
+                    continue
+
+                self.log(str(fabric_transit_details.get("sdaTransitSettings")))
+                self.log(str(item.get("sdaTransitSettings")))
+                if self.requires_update(fabric_transit_details.get("sdaTransitSettings"),
+                                        item.get("sdaTransitSettings"),
+                                        self.fabric_transits_obj_params):
+                    self.msg = "The SDA fabric transit config is not applied to the Cisco Catalyst Center."
+                    self.status = "failed"
+                    return self
+
+                fabric_transit_index += 1
+
+            self.log("Successfully validated SDA fabric transit(s).", "INFO")
+            self.result.get("response")[0].get("fabric_transits").update({"Validation": "Success"})
+
+        self.msg = "Successfully validated the SDA fabric transit(s)."
+        self.status = "success"
+        return self
+
+    def verify_diff_deleted(self, config):
+        """
+        Validating the Cisco Catalyst Center configuration with the playbook details
+        when state is deleted (delete).
+
+        Parameters:
+            config (dict) - Playbook details containing fabric transit configuration.
+        Returns:
+            self (object) - The current object with updated desired Fabric Transits information.
+        Description:
+            If the 'sda_fabric_transits' is available in the playbook, collect the fabric transit information
+            from the Cisco Catalyst Center.
+            Check if there is any transit with the name given in the playbook, set the msg by settings the
+            self.status as 'failed' and return self.
+        """
+
+        self.get_have(config)
+        self.log("Current State (have): {current_state}".format(current_state=self.have), "INFO")
+        self.log("Desired State (want): {requested_state}".format(requested_state=self.want), "INFO")
+        if config.get("sda_fabric_transits") is not None:
+            fabric_transit_index = 0
+            fabric_transit_details = self.have.get("fabric_transits")
+            for item in fabric_transit_details:
+                fabric_transit_exists = item.get("exists")
+                name = config.get("sda_fabric_transits")[fabric_transit_index].get("name")
+                if fabric_transit_exists:
+                    self.msg = (
+                        "The SDA fabric transit config '{name}' is not applied to"
```

Comment:
to make it clear 'is still present in' instead of 'is not applied to'

Reply:
As we are checking "fabric_transit_exists", 
The SDA fabric transit config '{name}' is still present in the Cisco Catalyst Center.